_pytest/tmpdir.py

```diff
@@ -43,7 +43,12 @@ def getbasetemp(self):
                     basetemp.remove()
                 basetemp.mkdir()
             else:
-                basetemp = py.path.local.make_numbered_dir(prefix='pytest-')
+                # use a sub-directory in the temproot to speed-up
+                # make_numbered_dir() call
+                rootdir = py.path.local.get_temproot().join('pytest')
+                rootdir.ensure(dir=1)
```

Comment:
sometimes people use pytest in different user accounts on the same machine and you would run into permission problems here.  we can probably use ``pytest-USERNAME`` as the base to disambiguate.